packages/esm-patient-attachments-app/src/camera-media-uploader/paste-image-url.component.tsx

```diff
@@ -0,0 +1,86 @@
+import React, { useState, useContext } from 'react';
+import { TextInput, Button, InlineNotification } from '@carbon/react';
+import CameraMediaUploaderContext from './camera-media-uploader-context.resources';
+import styles from './paste-image-url.scss';
+
+const PasteImageUrlComponent: React.FC = () => {
+  const [url, setUrl] = useState('');
+  const [error, setError] = useState(false);
+  const { setFilesToUpload } = useContext(CameraMediaUploaderContext);
+
+  const handleLoadImage = async () => {
+    try {
+      setError(false);
+
+      const response = await fetch(url, { mode: 'cors' });
```

Comment:
@dkayiwa @ibacher anything to be worried about from a security perspective? Also, do we need some URL validation logic here, or is that implicitly taken care of by the backend?